### ISSUE TITLE
zipcode_in_state for NY returns only one result

### ISSUE THREAD

**Issue by arielkaluzhny** [joke2k/faker#1060]
* Faker version: 2.0.2
* OS: macOS Mojave (version 10.14.6)

Brief summary of the issue goes here.

### Steps to reproduce
```python
import faker

fake = faker.Faker()
fake.zipcode_in_state("NY")
fake.zipcode_in_state("NY")
```

### Expected behavior
Results from the two calls to `zipcode_in_state` should return two different zipcodes.

### Actual behavior
`zipcode_in_state("NY")` always returns '06390'. (Other states return different zipcodes as expected)

**Comment by malefice** [comment]
I looked into this, and there is only one zipcode value listed for NY as shown:
https://github.com/joke2k/faker/blob/1eb101a81150b1926fb215f0b6dacd94e85886ef/faker/providers/address/en_US/__init__.py#L293

Unfortunately, I am not familiar with US zipcodes, so please consider submitting a pull request if you are.

**Comment by arielkaluzhny** [comment]
> I looked into this, and there is only one zipcode value listed for NY as shown:
> https://github.com/joke2k/faker/blob/1eb101a81150b1926fb215f0b6dacd94e85886ef/faker/providers/address/en_US/__init__.py#L293
> 
> Unfortunately, I am not familiar with US zipcodes, so please consider submitting a pull request if you are.

Yes thank you, I looked into this as well. I'm planning on submitting one soon!

**Comment by crd** [comment]
Should be considered closed by #1064

**Comment by fcurella** [comment]
Thank you @crd!
